_posts/2020-05-15-state-of-presto.md

```diff
@@ -32,4 +32,14 @@ Time: 11am PDT (San Francisco), 2pm EDT (New York), 7pm BST (London), 6pm UTC
 
 > ## [Register now!](https://www.starburstdata.com/webinar-state-of-presto/?utm_campaign=Webinar%20-%20State%20of%20Presto%20-%202020%20-%20May&utm_source=prestosql&utm_medium=blog)
 
-We look forward to many questions and a lively webinar.
\ No newline at end of file
+We look forward to many questions and a lively webinar.
+
+Update:
+
+We had a great event with lots of questions from the audience, taking us beyond
+the planned time frame. Check out the recording to learn more:
+
+<!-- <iframe width="420" height="315" -->
```

Comment:
We should be able to use the YouTube tag. See https://raw.githubusercontent.com/simpligility/prestosql.io/91c5142d8a6cd98067b57f06f61521d059e3c11e/_posts/2019-02-06-Presto-Community-Meeting.md

Reply:
The tag works correctly on mobile.

Reply:
Done